extensions/panache/panache-common/deployment/pom.xml

```diff
@@ -21,6 +21,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-panache-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-orm-deployment</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
`panache-common` is common to Hibernate and MongoDB, it should not depends on any Hibernate modules